packages/neuron-wallet/src/menu.ts

```diff
@@ -1,15 +1,27 @@
-import { app, shell, Menu, MenuItemConstructorOptions } from 'electron'
+import { app, shell, Menu, MenuItemConstructorOptions, dialog } from 'electron'
 
 const separator: MenuItemConstructorOptions = {
   type: 'separator',
 }
 
+const options = {
+  type: 'info',
+  title: app.getName(),
+  message: app.getName(),
+  detail: app.getVersion(),
+  buttons: ['OK'],
+  cancelId: 0,
+}
+
 const menuTemplate = [
   {
     label: app.getName(),
     submenu: [
       {
         role: 'about',
+        click: () => {
+          dialog.showMessageBox(options)
```

Comment:
Use `app.showAboutPanel()` to show the default about dialog on macOS.

Reply:
On macOS, `role: 'about'` priority is higher than `click:()`.
Win and Linux is the opposite.

Reply:
Good to know that! It's great! 👍